extensions/ui/org.eclipse.smarthome.ui.basic/src/main/java/org/eclipse/smarthome/ui/basic/internal/render/ChartRenderer.java

```diff
@@ -77,8 +77,13 @@ public boolean canRender(Widget w) {
             if (chartTheme != null) {
                 chartUrl += "&theme=" + chartTheme;
             }
-            // add timestamp to circumvent browser cache
-            String url = chartUrl + "&t=" + (new Date()).getTime();
+            String url;
+            if (!itemUIRegistry.getVisiblity(w)) {
+                url = "images/none.png";
```

Comment:
Feels a bit weird here, should be a static const maybe?

Reply:
Ok, why not.